set.mm

```diff
@@ -529946,6 +530091,155 @@ every nonzero prime ideal contains a prime element (this is a
       MNOPQRWHWIWJVHVIVJ $.
   $}
 
+  ${
+    fldextrspun.k $e |- K = ( L |`s F ) $.
+    fldextrspun.i $e |- I = ( L |`s G ) $.
+    fldextrspun.j $e |- J = ( L |`s H ) $.
+    fldextrspun.2 $e |- ( ph -> L e. Field ) $.
+    fldextrspun.3 $e |- ( ph -> F e. ( SubDRing ` I ) ) $.
+    fldextrspun.4 $e |- ( ph -> F e. ( SubDRing ` J ) ) $.
+    fldextrspun.5 $e |- ( ph -> G e. ( SubDRing ` L ) ) $.
+    fldextrspun.6 $e |- ( ph -> H e. ( SubDRing ` L ) ) $.
+    ${
+      fldextrspundglemul.7 $e |- ( ph -> ( J [:] K ) e. NN0 ) $.
+      fldextrspundglemul.1 $e |- E = ( L |`s ( L fldGen ( G u. H ) ) ) $.
+      $( Given two different field extensions ` I ` and ` J ` of a same field
```

Comment:
I think it's a bit misleading to write "different" here, since it is perfectly fine to take the same field extensions in this theorem. I also think it's better to write that these are field extensions of `K`, not `F` since the latter is a base set. Also perhaps one could use a "math textbook" field extension notation `I / K` and `J / K` etc. I propose to replace the comment by something like this:

"Given two finite extensions ` I / K ` and ` J / K ` of the same field ` K `, and the composite field ` E = IJ `, the degree of the extension ` E / K ` is at most the product of the field extension degrees of ` I / K ` and ` J / K `."

Reply:
Fixed in https://github.com/metamath/set.mm/pull/5063/commits/5ead495ed7ba8b74db5b24dc916a4443a949ff09